mitodl/open-discussions
# Improvements for RSS dialog editing

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1640 

#### What's this PR do?
Implements various changes relating to the dialog box for adding or updating rss widgets

#### How should this be manually tested?
Add or update an RSS widget and verify that things look good. In particular look at the unchecked items in the issue and make sure they all are addressed.

#### Screenshot
![screenshot from 2019-01-22 13-23-48](https://user-images.githubusercontent.com/863262/51556730-fcd7f200-1e48-11e9-8144-2e870b934707.png)

![screenshot from 2019-01-22 13-23-33](https://user-images.githubusercontent.com/863262/51556738-ffd2e280-1e48-11e9-8b1c-f099339a8d7a.png)
